modules/services/sharedtest/src/main/java/au/com/shiftyjelly/pocketcasts/sharedtest/InMemoryFeatureFlagRule.kt

```diff
@@ -0,0 +1,21 @@
+package au.com.shiftyjelly.pocketcasts.sharedtest
+
+import au.com.shiftyjelly.pocketcasts.utils.featureflag.FeatureFlag
+import au.com.shiftyjelly.pocketcasts.utils.featureflag.providers.InMemoryFeatureProvider
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+
+@ExperimentalCoroutinesApi
+class InMemoryFeatureFlagRule : TestWatcher() {
```

Comment:
Nice! 👍

Reply:
Thanks to you! https://github.com/Automattic/pocket-casts-android/pull/1874#discussion_r1499248252